azext_iot/central/params.py

```diff
@@ -42,6 +49,7 @@ def load_central_arguments(self, _):
         context.argument("app_id", options_list=["--app-id", "-n"], help="Target App.")
         context.argument("minimum_severity", arg_type=severity_type)
         context.argument("role", arg_type=role_type)
+        context.argument("sources", arg_type=sources)
```

Comment:
IMPT:

Since we are starting to add a bunch of commands, could you actually reorganize this file as part of this or next PR?

We should group all the command specific params to their parent group. See the following example:

```
with self.argument_context("iot central app validate-messages") as context:
        context.argument("timeout", arg_type=event_timeout_type)
        context.argument("properties", arg_type=event_msg_prop_type)
        context.argument("style", arg_type=style_type)
        context.argument(
            "duration",
            options_list=["--duration", "--dr"],
            type=int,
            help="Maximum duration to receive messages from target device before terminating connection."
            "Use 0 for infinity.",
        )
        context.argument(
            "max_messages",
            options_list=["--max-messages", "--mm"],
            type=int,
            help="Maximum number of messages to recieve from target device before terminating connection."
            "Use 0 for infinity.",
        )
```

For example, if we wanted to do this with CDE:

```
with self.argument_context("iot central app cde") as context:
        context.argument(
            "ep_type",
            options_list=["--ep-type", "-t"],
            arg_type=get_enum_type(EndpointType),
            help="Type of endpoint where exported data should be sent to. ",
        )
```

If we push this till later - we run a bigger risk of having clashing param names/arguments